remember.py

```diff
@@ -0,0 +1,30 @@
+"""
+Requieremenrs : PIL Package
+
+This program gets path to a file, then is takes the row number of each black pixel,
+then uses the chr() function to convert it to a letter.
+@param : path - path to a png file
+@return : string
+"""
+
+from PIL import Image
+
+
+def decode_png(path):
```

Comment:
Use type hints and return types -docum
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/